_articles/getting-paid.md

```diff
@@ -104,11 +104,16 @@ If you can't convince your current employer to prioritize open source work, cons
 
 Projects that originated at a large company, such as [Go](https://github.com/golang) or [React](https://github.com/facebook/react), will also likely employ people to work on open source.
 
-Finally, depending on your personal circumstances, you can try raising money independently to fund your open source work. For example:
+Depending on your personal circumstances, you can try raising money independently to fund your open source work. For example:
 
 * @gaearon funded his work on [Redux](https://github.com/reactjs/redux) through a [Patreon crowdfunding campaign](https://redux.js.org/)
 * @andrewgodwin funded work on Django schema migrations [through a Kickstarter campaign](https://www.kickstarter.com/projects/andrewgodwin/schema-migrations-for-django)
 
+Finally, there are also open source projects which put bounties on issues that you might consider helping with.  Many of the organizations funding their projects also use bounty platforms for recruiting. For Example: 
+
+* @ConnorChristie was able to get paid for [helping](https://github.com/MARKETProtocol/MARKET.js/issues/14) @MARKETProtocol work on their javascript library [through a bounty on gitcoin](https://gitcoin.co/).
```

Comment:
Is the a specific page on gitcoin to link to so that the link text corresponds to the link?

Reply:
I'm not sure what you mean by text corresponding... maybe the about page here? https://gitcoin.co/about